README.md

```diff
@@ -23,10 +23,12 @@ Subgraph is used to track the state of metadata by indexing events logged throug
 
 Content of the metadata will not be verified as the registry is designed to be generic and can accept any metadata. It is the Dapp's responsibility to verify metadata.
 
-See subgraph entity definitions: [/subgraph/schema.graphql](./subgraph/schema.graphql).
-
+See subgraph entity definitions: [/packages/subgraph/schema.graphql](./packages/subgraph/schema.graphql).
```

Comment:
generate UML from schema for the docs: 
https://github.com/sheerun/graphqlviz or
https://github.com/APIs-guru/graphql-voyager

Reply:
added UML as requested.